userdocs/src/components/kubernetes.md

```diff
@@ -36,7 +36,7 @@ spec:
       name: admin_password
 ```
 
-When this definition is applied to the cluster with `kubectl apply -f {secret.yaml}` it will result in a Kubernets secret being created.
+When this definition is applied to the cluster with `kubectl apply -f {secret.yaml}`, or if you deploy your pods to a namespace: `kubectl apply -f {secret-yaml} --namespace {your-namespace}`, it will result in a Kubernets secret being created.
```

Comment:
You always apply a pod (or any Kubernetes resource) to a namespace, it's not possible not to.

The reason `--namespace` often is omitted in docs, is that if you do kubectl apply <something> (without --namespace), it is going to apply to the current namespace. So it's not necessary to add `--namespace` to every command. However, not all users may know this, so it doesn't hurt to put it here. If we have many kubectl commands, though, I would rather mention how the user can set the active namespace in the start of the article, so we can avoid the `--namespace` argument everywhere.

Just FYI, the active namespace can be set like this:

```bash
kubectl config current-context // output example: my-context
kubectl config set-context my-context --namespace=mystuff
kubectl config use-context my-context
```

But a simpler alternative is to install for instance https://github.com/ahmetb/kubectx

Anyway, I suggest we write something like this:

When this definition is applied to the cluster with `kubectl apply -f {secret.yaml}`, or if you apply the definition to a specific namespace: `kubectl apply -f {secret-yaml} --namespace {your-namespace}`, it will result in a Kubernets secret being created.